src/Classes/Loader.ts

```diff
@@ -19,6 +19,10 @@ export class Loader {
         return Loader.dom()[index];
     }
 
+    public static loadFromArray(tagId: string, params: Array<IOptions>, index?: number): Container | undefined {
+        return Loader.load(tagId, params[(index === undefined || index < 0 || index >= params.length) ? Math.floor(Math.random() * params.length) : index]);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5086765)